tests/bdd/features/rebuild/log-based-rebuild.feature

```diff
@@ -21,4 +21,12 @@ Feature: Partial Rebuild
     And the replica is online again within the timed-wait period
     Then log-based rebuild starts after some time
     But the node hosting rebuilding replica crashes permanently
-    Then a full rebuild starts after some time
\ No newline at end of file
+    Then a full rebuild starts after some time
+
+  Scenario: Volume target is unreachable during log based rebuild
+    Given a volume with three replicas, filled with user data
+    When a child becomes faulted
```

Comment:
It's not very clear which two children are involved here. 
```suggestion
    When a non-local child becomes faulted
```

Reply:
Yes I think this is required, otherwise at the last then statement only one rebuild would happen?

Reply:
Done